### ISSUE TITLE
Pages menu, scroll with pages 

### ISSUE THREAD

**Issue by Intranox** [flxzt/rnote#639]
I really like this software and it really includes a lot of things. One feature that could be useful would be to have the ability to have a menu where you can jump directly to a page or enter the page number and go directly to one of them. It depends on the use sometimes documents get quite long and jumping to specific points in the document through the pages with preview could help a lot

**Comment by vBazilevich** [comment]
Another useful related feature: pages re-ordering

**Comment by Intranox** [comment]
> Another useful related feature: pages re-ordering

Yes, this is a good idea. Another interesting idea could be to add a page between two already created pages
